maps.md

```diff
@@ -414,8 +414,6 @@ func (d Dictionary) Add(word, definition string) error {
 	default:
 		return err
 	}
-
-	return nil
```

Comment:
I don't think that this code not reachable

if `word` is in the dictionary we enter the `case ErrNotFound` switch branch which adds the entry to the dictionary, subsequently the code continues and at the end it returns `nil` since there were no errors... 

or am I missing something? :confused: